crates/sui-indexer-alt/src/db.rs

```diff
@@ -156,3 +170,48 @@ pub async fn reset_database(
     }
     Ok(())
 }
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::tempdb::TempDb;
+    use diesel::prelude::QueryableByName;
+    use diesel_async::RunQueryDsl;
+
+    #[derive(QueryableByName)]
+    struct CountResult {
+        #[diesel(sql_type = diesel::sql_types::BigInt)]
+        cnt: i64,
+    }
```

Comment:
I think this may be unnecessary -- i.e. you could `get_result::<i64>` directly.

Reply:
Ah, because the table is also created dynamically it doesn't know what type the column is -- thanks for clarifying.